src/main/resources/db/migration/V025__Update_ocrData_column.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE scannable_items
+  DROP COLUMN ocrData;
+
+ALTER TABLE scannable_items
+  ADD COLUMN ocrData JSON null
```

Comment:
It took my a while to realise `ocrData` column is currently not used.
I think this migration deserves a comment, at least on the PR.

Reply:
I'll add a comment. 
We want to do this in 2 PRs. The first PR to add the field and the second PR will have the migration script to use to drop the existing `ocrDataJson` and use the field `ocrData`.

Reply:
Dropping ocrDataJson alone will need to be done in a separate deployment, as the last change.

Reply:
👍